apps/i18n/panels/en_us.json

```diff
@@ -0,0 +1,4 @@
+{
+  "continue": "Continue",
+  "next": "Next"
```

Comment:
Done.  It was tempting to leave the loc support there so that the app could serve as another example of bringing up an entire Lab2 app, but it is certainly cleaner without that support since it's not currently needed.  The [removal commit](https://github.com/code-dot-org/code-dot-org/pull/55758/commits/cc918ab43d7a630e94ac5eec96a559c5bacf2f5f) might be useful some day.

Reply:
Yeah cool makes sense, and I'd also like to see if we can reduce the number of loc-related files needed to bring up a new lab